docs/MigrationGuide.md

```diff
@@ -0,0 +1,58 @@
+# Migration guide from POMI (nri-prometheus) to newrelic-prometheus
```

Comment:
Does it make sense to specify also some of the good new features or should we do it just in the docs?

Es: More config option, better scalability, etc